src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { TodosFilter } from '../TodosFilter';
+
+type Props = {
+  notCompletedTodosLength: number;
+  hasCompletedTodo: boolean;
+  onDeleteCompletedTodos: () => Promise<void>
+};
+
+export const Footer :React.FC<Props> = ({
+  notCompletedTodosLength,
+  hasCompletedTodo,
+  onDeleteCompletedTodos,
+}) => {
+  return (
+    <footer className="todoapp__footer" data-cy="Footer">
+      <span className="todo-count" data-cy="TodosCounter">
+        {`${notCompletedTodosLength} items left`}
```

Comment:
When the number is one, there should be no plural message:
<img width="574" alt="image" src="https://github.com/mate-academy/react_todo-app-with-api/assets/54860641/a33a7e4b-1240-4778-9f86-9d5d7f81b152">

Reply:
😅😅
![image](https://github.com/mate-academy/react_todo-app-with-api/assets/91376823/d5f0d1c5-d48e-484d-9852-82b9e17f3dab)